libpod/options.go

```diff
@@ -186,6 +186,19 @@ func WithHooksDir(hooksDir string) RuntimeOption {
 	}
 }
 
+// WithDefaultMountsFile sets the file to look at fot secret mounts
+// Note we are not saving this in the database as it is for testing purposes only
+func WithDefaultMountsFile(mountsFile string) RuntimeOption {
```

Comment:
if this is for secrets only, should 'Secrets' be in the function name somehow?

Reply:
`WithSecretMountsFile()` sounds fine to me

Reply:
went with DefaultMountsFile because the file is not restricted to only secrets, the user can add anything they want mounted into the container there.
I can still change it to be WithSecretsMountsFile if we really want it to be that. But the defaults are called `OverrideMountsFile and DefaultMountsFille` in secrets.go.

Reply:
SGTM

Reply:
Modified the comment, hope it is better.